kong-1.4.2-0.rockspec

```diff
@@ -36,7 +36,7 @@ dependencies = {
   "lua-resty-cookie == 0.1.0",
   "lua-resty-mlcache == 2.4.0",
   "lua-messagepack == 0.5.2",
-  "lua-resty-openssl == 0.3.0",
+  "lua-resty-openssl == 0.4.1",
```

Comment:
I see you added new dependency, but the goal was to remove `luaossl` dependency. Is that still the case or why wasn't `"luaossl == 20190731"` removed?

Reply:
Thanks for catching that, I missed it = ) Adding it as a new commit and squashed the suggestion commit.